web_widget_slick/static/tests/js/web_widget_slick.js

```diff
@@ -0,0 +1,83 @@
+/* Copyright 2017 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html). */
+
+odoo.define_section('web_widget_slick', ['web.core', 'web.form_common'],
```

Comment:
I typically suffix my tours with `.tour`, but that's stylistic so feel free to leave this.

Reply:
These are just unit tests, not a tour. 😄